app/assets/stylesheets/components/_carousel_images.scss

```diff
@@ -0,0 +1,9 @@
+.carousel {
+  display: flex;
+  width: 100%;
+
+  justify-content: center;
+}
+.d-block {
+  height: 200px;
+}
```

Comment:
What do you think about adding property object-fit:cover; to avoid distortions in the carousel images.